src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -195,11 +195,13 @@
 				:aria-label="t('spreed', 'Lower hand (R)')"
 				@shortkey="toggleHandRaised"
 				@click.stop="toggleHandRaised">
+				<!-- The following icon is much bigger than all the others
+						so we reduce its size -->
 				<HandBackLeft
-					:size="20"
+					decorative
 					title=""
-					fill-color="#ffffff"
-					decorative />
+					size="18px"
```

Comment:
💥 
![Bildschirmfoto von 2022-01-12 14-04-45](https://user-images.githubusercontent.com/213943/149146072-3ea2bb7d-4d1a-45fb-8297-762bcb7602c7.png)

```suggestion
					size="18"
```

You didn't try this?